package.json

```diff
@@ -59,19 +85,22 @@
 			},
 			{
 				"command": "camelk.integrations.remove",
-				"title": "Remove Integration"
+				"title": "Remove Camel-K Integration"
+			},
+			{
+				"command": "camelk.integrations.log",
+				"title": "View log for Camel-K Integration"
```

Comment:
in Kubernetes extension, the naming used is:
"Show logs" if provide value at a single point of time
"Follow logs" to have logs updated.

I think it would be nice to reuse the same naming

Reply:
Will change to "Follow log for Camel-K Integration"